Source/ChocolateyGui.Common.Windows/Views/PackageView.xaml

```diff
@@ -60,6 +60,13 @@
                     VerticalAlignment="Stretch"
                     Background="{DynamicResource MahApps.Brushes.Accent4}">
                 <StackPanel Orientation="Horizontal" HorizontalAlignment="Right" Margin="0,10,20,10">
+                        <Button Padding="10" Margin="5 0"
+                            Command="{commands:DataContextCommandAdapter ShowArguments}">
+                            <StackPanel Orientation="Horizontal">
+                                <iconPacks:PackIconEntypo Kind="Info" Margin="0 0 5 0 " VerticalAlignment="Center" />
+                                <TextBlock Text="Package Arguments" FontSize="16"/>

```

Comment:
We need to localize this string, in the Resource.resx file. Let me know if you are not clear on how this can be done, and we can have a chat about it.